tests/DocumentationTest.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Support\Str;
+use PHPUnit\Framework\TestCase;
+
+class DocumentationTest extends TestCase
+{
+    public function testIfDocumentationContainsAllPublicFunctions(): void
+    {
+        $functions = $this->getPublicFunctions();
+
+        foreach ($functions as $class => $methods) {
+            $trait = explode("\\", $class );
+            $fileName = Str::slug(end($trait)) . ".html";
+            $filePath = "docs/elements/$fileName";
+
+            if (!file_exists($filePath)) {
+                if (str_ends_with($filePath, "httprequest.html")) {
```

Comment:
Why not to rename files to be unified?

Reply:
We prefer one Http file in documentation, but separating Into HttpRequest and HttpResponse might be easier to find what user is looking for. Should we separate it?

Reply:
Then maybe some structure instead of four ifs would be better?